rsconnect_jupyter/static/rsconnect.js

```diff
@@ -285,9 +285,16 @@ define([
                             if (result['last_status'] != lastStatus) {
                                 lastStatus = result['lastStatus'];
                                 var output = result['status'].join('\n');
+
+                                var elem = $log[0];
```

Comment:
```suggestion
                                var elem = $log.get(0);
```

Also consider using a symbol name that describes the element's purpose.